src/components/IconButton/IconButton.tsx

```diff
@@ -1,28 +1,66 @@
 import React, { FC, JSXElementConstructor } from 'react';
 
 import classNames from 'classnames';
-import styles from './IconButton.module.scss';
 import { IconProps } from '../Icons/Icons.types';
 
+import './IconButton.scss';
+
 export interface IconButtonProperties {
   className?: string;
   onClick: () => void;
 
   Icon: JSXElementConstructor<IconProps>;
   label?: string;
-  size?: string | number;
+  size?: string | number | 'large' | 'small' | 'x-small';
```

Comment:
Nice, we'll have to update where this is used in zApps so we can remove `string|number`. One for later!

Reply:
maybe we can create a common `enum` as well (just to avoid repeating using raw string) ?